python/device/device.py

```diff
@@ -610,31 +583,10 @@ def send_metrics_to_azure_monitor(self, props):
         Send metrics to azure monitor, based on the reported properties that we probably just
         sent to the hub
         """
-        # we don't record session_metrics to azure monitor
-
-        system_health_metrics = props[Fields.Reported.SYSTEM_HEALTH_METRICS]
-        self.reporter.set_process_cpu_percent(system_health_metrics["processCpuPercent"])
-        self.reporter.set_process_working_set(system_health_metrics["processWorkingSet"])
-        self.reporter.set_process_bytes_in_all_heaps(
-            system_health_metrics["processBytesInAllHeaps"]
-        )
-        self.reporter.set_process_private_bytes(system_health_metrics["processPrivateBytes"])
-        self.reporter.set_process_working_set_private(system_health_metrics["processCpuPercent"])
-
-        test_metrics = props[Fields.Reported.TEST_METRICS]
-        self.reporter.set_send_message_count_sent(test_metrics["sendMessageCountSent"])
-        self.reporter.set_send_message_count_received_by_service_app(
-            test_metrics["sendMessageCountReceivedByServiceApp"]
-        )
-        self.reporter.set_send_message_count_in_backlog(test_metrics["sendMessageCountInBacklog"])
-        self.reporter.set_send_message_count_unacked(test_metrics["sendMessageCountUnacked"])
-        self.reporter.set_send_message_count_not_received_by_service_app(
-            test_metrics["sendMessageCountNotReceivedByServiceApp"]
-        )
-        self.reporter.set_receive_message_count_received(
-            test_metrics["receiveMessageCountReceived"]
-        )
-        self.reporter.set_receive_message_count_missing(test_metrics["receiveMessageCountMissing"])
+        # we don't record session_metrics to azure monitor because they're all about time and i
```

Comment:
Can you re-read this sentence and see if it makes sense to you? It seems the end of this line doesn't continue with the next in a way that makes sense to me.

Reply:
I remember feeling awkward when I was writing that.  I'll try again :)